### PR TITLE
Move python modules from example/common/python to common/python

### PR DESCRIPTION
**Pull request by manju956**
All python modules are moved from example/common/python to common/python.
Also, all crypto related modules are grouped and placed under crypto_utils module and
crypto util functions are removed from utility.py and added in newly created crypto_utility.py

Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: client_sdk/avalon_client_sdk/work_order/work_order_params.py [hunk 1]
```diff
@@ -15,9 +15,9 @@
 import json
 import logging
 
-import crypto.crypto as crypto
-import utility.utility as utility
-import utility.signature as signature
+import crypto_utils.crypto.crypto as crypto
+import crypto_utils.crypto_utility as utility
```
Comment (manojgop) [inline client_sdk/avalon_client_sdk/work_order/work_order_params.py]: as utility -> as crypto_utility

File: common/python/crypto_utils/crypto_utility.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Intel Corporation
+# Copyright 2020 Intel Corporation
```
Comment (manojgop) [inline common/python/crypto_utils/crypto_utility.py]: Keep it as 2018. This is just file rename. Old functions in this file still hold copyright from 2018 ?

File: common/python/crypto_utils/crypto_utility.py [hunk 3]
```diff
@@ -12,51 +12,14 @@
...
-    error_response["jsonrpc"] = "2.0"
-    error_response["id"] = jrpc_id
-    error_response["error"] = {}
-    error_response["error"]["code"] = code
-    error_response["error"]["message"] = message
-    return error_response
-
-
-# -----------------------------------------------------------------------------
-def strip_begin_end_key(key):
```
Comment (manojgop) [inline common/python/crypto_utils/crypto_utility.py]: This function can be kept in crtypo_utils as its doing operation specifically for public key.  Thsi function could be renamed as strip_begin_end_public_key

File: common/python/setup.py [hunk 4]
```diff
@@ -109,7 +109,7 @@
 
 # -----------------------------------------------------------------
 setup(
-    name='tcf_examples_common',
+    name='avalon_common',
     version=version,
     description='Common library',
     author='Intel',
```
Comment (manojgop) [inline common/python/setup.py]: author='Hyperledger Avalon',
url='https://github.com/hyperledger/avalon',

File: common/python/utility/utility.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+import os
+import utility.file_utils as putils
+import utility.hex_utils as hex_utils
+import crypto_utils.crypto.crypto as crypto
+import config.config as pconfig
+import logging
+
+logger = logging.getLogger(__name__)
+
+def create_error_response(code, jrpc_id, message):
```
Comment (manojgop) [inline common/python/utility/utility.py]: This should be moved to a different error handling file. Or create a json_utils.py

File: common/python/crypto_utils/crypto_utility.py (hunk 2 above)
Comment (manju956) [reply common/python/crypto_utils/crypto_utility.py]: I have retained utility.py and created this new file to group all crypto util functions. Hence copyright is updated to latest.

File: client_sdk/avalon_client_sdk/work_order/work_order_params.py (hunk 1 above)
Comment (manju956) [reply client_sdk/avalon_client_sdk/work_order/work_order_params.py]: Done

File: common/python/crypto_utils/crypto_utility.py (hunk 3 above)
Comment (manju956) [reply common/python/crypto_utils/crypto_utility.py]: Ok

File: common/python/setup.py (hunk 4 above)
Comment (manju956) [reply common/python/setup.py]: Done

File: common/python/utility/utility.py (hunk 5 above)
Comment (manju956) [reply common/python/utility/utility.py]: I removed all the unused functions in utility.py and renamed this file to jrpc_utils.py as the file now has only one jrpc utility function and nothing else.

File: common/python/utility/jrpc_utility.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Intel Corporation
```
Comment (Ram-srini) [inline common/python/utility/jrpc_utility.py]: s/2018/2020

Comment (manju956) [reply common/python/utility/jrpc_utility.py]: its an old file but renamed.

**Pull request merged**